### DISCUSSION TITLE
saf-training/courses/profile-dev-test/02

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#61] [Ideas]
# saf-training/courses/profile-dev-test/02

Repository Organization The repository and profile are organized into two primary branches: main and TBD. The repository has a set of tags representing iterative releases of the...

https://mitre.github.io/saf-training/courses/profile-dev-test/02.html

**Comment by aaronlippold** [comment]
Let's see if we can close out our language on the first paragraph here to standardize it.
